app/settings/settings.json

```
@@ -25,25 +25,45 @@
                 "bg": "#0e0f0f",
                 "fg": "#f5f6fa",
                 "activebackground": "#635f5f",
-                "activeforeground": "#000000"
+                "activeforeground": "#000000",
+                "borderwidth": 0,
+                "highlightthickness": 0,
+                "width": 6,
+                "height": 2,
+                "font": "Arial 14 bold"
             },
             "BTN_NUMERICO": {
                 "bg": "#050505",
                 "fg": "#f5f6fa",
                 "activebackground": "#635f5f",
-                "activeforeground": "#000000"
+                "activeforeground": "#000000",
+                "borderwidth": 0,
+                "highlightthickness": 0,
+                "width": 6,
+                "height": 2,
+                "font": "Arial 14 bold"
             },
             "BTN_OPERADOR": {
                 "bg": "#0e0f0f",
                 "fg": "#f5f6fa",
                 "activebackground": "#0097e6",
-                "activeforeground": "#000000"
+                "activeforeground": "#000000",
+                "borderwidth": 0,
+                "highlightthickness": 0,
+                "width": 6,
+                "height": 2,
+                "font": "Arial 14 bold"
             },
             "BTN_CLEAR": {
                 "bg": "#0e0f0f",
                 "fg": "#f5f6fa",
                 "activebackground": "#d63031",
-                "activeforeground": "#000000"
+                "activeforeground": "#000000",
+                "borderwidth": 0,
+                "highlightthickness": 0,
+                "width": 6,
+                "height": 2,
+                "font": "Arial 14 bold"
```

Mantenha o destacado como verde e remova o destacado em vermelho:

```suggestion
                "activeforeground": "#000000"
```